dev_docs/key_concepts/feature_privileges.mdx

```diff
@@ -0,0 +1,435 @@
+- **Type**: string
+- **Example**: `"My Feature"`
+
+#### description (optional)
+
+- **Description**: An optional description that will appear as subtext underneath the feature name.
+- **Type**: string
+- **Example**: `"This is a feature that allows users to perform advanced actions."`
+
+#### category
+
+- **Description**: The category for this feature. This will be used to organize the list of features for display within the Spaces and Roles management screens.
+- **Type**: AppCategory (enum)
+- **Example**: `AppCategory.Security`
+
+#### order (optional)
+
+- **Description**: An ordinal used to sort features relative to one another for display.
+- **Type**: number
+- **Example**: `10`
+
+#### excludeFromBasePrivileges (optional)
+
+- **Description**: Whether or not this feature should be excluded from the base privileges. This is primarily helpful when migrating applications with a "legacy" privileges model to use Kibana privileges.
+- **Type**: boolean
+- **Example**: `false`
+
+#### minimumLicense (optional)
+
+- **Description**: Optional minimum supported license. If omitted, all licenses are allowed. This does not restrict access to your feature based on license. Its only purpose is to inform the space and roles UIs on which features to display.
+- **Type**: LicenseType (enum)
+- **Example**: `LicenseType.Gold`
+
+#### app
+
+- **Description**: An array of app IDs that are enabled when this feature is enabled. Apps specified here will automatically cascade to the privileges defined below, unless specified differently there.
+- **Type**: string[]
+- **Example**: `["app1", "app2"]`
+
+#### management (optional)
+
+- **Description**: If this feature includes management sections, you can specify them here to control visibility of those pages based on the current space.
+- **Type**: Object
+- **Example**:
+  ```js
+  management: {
+    kibana: ["settings", "indices"],
+    myApp: ["config"]
+  }
+  ```
+
+#### catalogue (optional)
+
+- **Description**: If this feature includes a catalogue entry, you can specify them here to control visibility based on the current space.
+- **Type**: string[]
+- **Example**: `["cat1", "cat2"]`
+
+#### alerting (optional)
+
+- **Description**: If your feature grants access to specific Alert Types, you can specify them here to control visibility based on the current space.
+- **Type**: string[]
+- **Example**: `["alertType1", "alertType2"]`
+
+#### cases (optional)
+
+- **Description**: If your feature grants access to specific case types, you can specify them here to control visibility based on the current space.
+- **Type**: string[]
+- **Example**: `["caseType1", "caseType2"]`
+
+#### privileges
+
+- **Description**: Feature privilege definition. These are used to control what access the read/all roles will have when enabled for your application.
+- **Type**: Object
+- **Example**:
+  ```js
+  privileges: {
+    all: {...}, // expanded below
```

Comment:
Can we link to the interface description below? It doesn't have to be on this line, maybe in the description? As it stands it's not super clear what this is referencing

Reply:
@kc13greiner Good point! I added a hyperlink to the interface details below